Library/Formula/homebank.rb

```diff
@@ -26,7 +26,7 @@ def install
     args << "--with-ofx" if build.with? 'libofx'
 
     system "./configure", *args
-    system "chmod +x ./install-sh"
+    chmod("+x", "install-sh")
```

Comment:
Personally I'd change it to `u+x`, never been much a fan of the octal numbers.

Anyways, this is not why I does fail the process for mountain lion.